pynbody/bridge/_bridge.pyx

```diff
@@ -50,8 +50,7 @@ def bridge(npc.ndarray[integral_1, ndim=1] iord_to,
             output_index[i] = 0
             found_match[i] = 0 # false
 
-    return (output_index, found_match.astype(np.bool))
-
+    return (output_index, found_match.astype(bool))
```

Comment:
Note: `np.bool` and `np.int` have been deprecated in favour of `bool` and `int` respectively.
See https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated.